client/rhel/spacewalk-client-tools/src/up2date_client/rhnreg_constants.py

```diff
@@ -19,77 +19,76 @@
 WHY_REGISTER_SEC      = _("Security & Updates:")
 WHY_REGISTER_DLD      = _("Downloads & Upgrades:")
 WHY_REGISTER_SUPP     = _("Support:")
 WHY_REGISTER_COMP     = _("Compliance:")
 WHY_REGISTER_SEC_TXT  = _("Receive the latest software updates, including security updates, keeping this "
-                          "Red Hat Enterprise Linux system updated and secure.")
-WHY_REGISTER_DLD_TXT  = _("Download installation images for Red Hat Enterprise Linux releases, "
+                          "system updated and secure.")
+WHY_REGISTER_DLD_TXT  = _("Download installation images for system releases, "
                           "including new releases.")
-WHY_REGISTER_SUPP_TXT = _("Access to the technical support experts at Red Hat or Red Hat's partners for help "
+WHY_REGISTER_SUPP_TXT = _("Access to the technical support experts for help "
                           "with any issues you might encounter with this system.")
 WHY_REGISTER_COMP_TXT = _("Stay in compliance with your subscription agreement "
                           "and manage subscriptions "
                           "for systems connected to your account.")
-WHY_REGISTER_TIP      = _("Tip: Red Hat values your privacy: "
-                          "http://www.redhat.com/legal/privacy_statement.html")
+WHY_REGISTER_TIP      = _("Tip: SUSE values your privacy: "
+                          "https://www.suse.com/company/legal/")
 BACK_REGISTER         = _("Take me back to the registration")
 
 # Confirm Quit Window
 CONFIRM_QUIT           = _("Software Update Not Set Up")
-CONFIRM_QUIT_SURE       = _("Are you sure you don't want to connect your system to Red Hat Satellite? "
-                           "You'll miss out on the benefits of a Red Hat Enterprise Linux subscription:\n")
+CONFIRM_QUIT_SURE       = _("Are you sure you don't want to connect your system to Uyuni? "
+                           "You'll miss out on the benefits of a SLES/RHEL subscription:\n")
 CONFIRM_QUIT_WILLNOT       = _("You will not be able to take advantage of these subscription privileges without connecting "
-                           "your system to Red Hat Satellite.\n")
+                           "your system to Uyuni.\n")
 CONTINUE_REGISTERING   = _("Take me back to the setup process.")
 REGISTER_LATER2        = _("I'll register later.")
 
 # Info Window
-REGISTER_WINDOW   = _("Red Hat Account")
-LOGIN_PROMPT      = _("Please enter your login information for the %s Red "
-                    "Hat Network Satellite:\n\n")
-HOSTED_LOGIN      = _("Red Hat Login:")
+REGISTER_WINDOW   = _("Uyuni Account")
+LOGIN_PROMPT      = _("Please enter your login information for the %s Uyuni:\n\n")
+HOSTED_LOGIN      = _("Uyuni Login:")
 LOGIN             = _("Login:")
 PASSWORD          = _("Password:")
 LOGIN_TIP         = _("Tip: Forgot your login or password?  Contact your "
-                      "Satellite's Organization Administrator.")
+                      "Uyuni Organization Administrator.")
```

Comment:
```suggestion
                      "Organization Administrator.")
```

Downstream users may not know who the "Uyuni Organization Administrator" is. They will know about the "SUSE Manager Organization Administrator".

"Organization Administrator" is not good but IMHO it's less bad than "Uyuni Organization Administrator".